src/app/ir/Custom.ts

```diff
@@ -22,7 +24,8 @@ class Custom {
     if (parsedFuncs instanceof Error) {
       console.error(parsedFuncs);
       return 'CUSTOM_NODE_PARSE_ERROR';
-    } if (parsedFuncs.length > 0) {
+    }
+    if (parsedFuncs.length > 0) {
```

Comment:
This will need to be added to the other `ModelCustom`, `DataCustom` and `OverviewCustom` as well.